appveyor.yml

```diff
@@ -12,6 +12,7 @@ configuration:
 
```

Comment:
"No artifacts found matching '*examples*.zip' path" - look at the end of appveyor log